hadoop-hdfs-project/hadoop-hdfs-httpfs/src/test/java/org/apache/hadoop/fs/http/client/BaseTestHttpFSWith.java

```diff
@@ -2056,6 +2060,27 @@ private void testGetSnapshotDiffListing() throws Exception {
     }
   }
 
+  private void testGetFileLinkStatus() throws Exception {
+    if (isLocalFS()) {
+      // do not test the the symlink for local FS.
```

Comment:
nit:
two times ``the the``

Reply:
Thank you so much for your assistance in reviewing the code! I truly appreciate your valuable feedback and will make the necessary modifications.